docs/source/differentiation/background.rst

```diff
@@ -0,0 +1,77 @@
+Background and Methods
+======================
+
+In this section we will explain the mathematical background of forward, backward and
+central differences. The main ideas in this chapter are taken from this [Dennis]_ and
+Schneidler. Our toolkit is developed for multivariate functions but can of
```

Comment:
reference to the Schneidler book (?) is missing.

Reply:
That took time. It's just one book, but not by Dennis and Schneidler, but Dennis and Schnabel!